docs/output.md

```diff
@@ -37,6 +54,173 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 
 > **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality.
 
+### fastp
+
+fastp can automatically detect adapter sequences for Illumina data.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `fastp`
+
+</details>
+
+### AdapterRemoval
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `adapterremoval`
+
+</details>
```

Comment:
```suggestion
[AdapterRemoval](https://adapterremoval.readthedocs.io/en/stable/) searches for and removes remnant adapter sequences from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases from the 3' end of reads following adapter removal. It is popular in the field of palaeogenomics. The output logs are stored in the results folder, and as a part of the MultiQC report.

<details markdown="1">
<summary>Output files</summary>

- `QC_shortreads/adapterremoval/[sample]/`
  - `<sample_id>.settings`: AdapterRemoval log file containing general adapter removal, read trimming and merging statistics
  - `<sample_id>.collapsed.fastq.gz` - read-pairs that merged and did not undergo trimming (only when  `--shortread_qc_mergepairs` supplied)
  - `<sample_id>.collapsed.truncated.fastq.gz`  - read-pairs that merged underwent quality trimming  (only when  `--shortread_qc_mergepairs` supplied)
  - `<sample_id>.pair1.truncated.fastq.gz`  - read 1 of pairs that underwent quality trimming
  - `<sample_id>.pair2.truncated.fastq.gz`  - read 2 of pairs that underwent quality trimming  (and could not merge if  `--shortread_qc_mergepairs` supplied)
  - `<sample_id>.singleton.truncated.fastq.gz` - orphaned read pairs where one of the pair was discarded
  - `<sample_id>.discard.fastq.gz` - reads that were discarded due to length or quality filtering

</details>
```

By default nf-core/taxprofiler will only provide the `.settings` file if AdapterRemoval is selected. To save processed reads for use in other analyses provide ` --save_preprocessed_reads` . If this is selected, you may recieve different combinations of FASTQ files for eac hsample depending on the input types - e.g. whether you have merged or not, or if you're supplying both single- and paired-end reads.